### DISCUSSION TITLE
Shouldn't it use {config.appName}

### DISCUSSION THREAD

**Discussion by ddiesenreither** [ionic-team/capacitor#5102] [Q&A]
Hi,

https://github.com/ionic-team/capacitor/blob/9a6e74e2eee1f8fd09f915fe7651b09df52e96b5/cli/src/ios/run.ts

on line 49

I get the error:
 ✖ Deploying App.app to 00008101-000315E21480001E - failed!
[capacitor] [error] ERR_UNKNOWN: Path '/Users/me/WebstormProjects/apkkp/apkkp-mobile/ios/DerivedData/00008101-000315E21480001E/Build/Products/Debug-iphoneos/App.app' not found

Because there is an APK-KP.app but not the App.app as used.

config.appName is 'APK-KP' but config.ios.scheme is per default 'App' so it is looking for 'App.app' which is wrong.

I am using Xcode 13.

**Comment by allidoisace** [comment]
I am having a similar issue:

```
> capacitor run ios --no-sync --target 00008027-001E40590A6A402E
[capacitor] ✖ Running xcodebuild - failed!
[capacitor] [error] Command line invocation:
[capacitor]         /Applications/Xcode.app/Contents/Developer/usr/bin/xcodebuild -workspace App.xcworkspace -scheme App -configuration Debug -destination id=00008027-001E40590A6A402E -derivedDataPath .../Documents/my-app/app/ios/DerivedData/00008027-001E40590A6A402E
[capacitor]         
[capacitor]         User defaults from command line:
[capacitor]         IDEDerivedDataPathOverride = .../Documents/my-app/app/ios/DerivedData/00008027-001E40590A6A402E
[capacitor]         IDEPackageSupportUseBuiltinSCM = YES
[capacitor]         
[capacitor]         xcodebuild: error: The workspace named "App" does not contain a scheme named "App". The "-list" option can be used to find the names of the schemes in the workspace.
```

**Comment by allidoisace** [comment]
@ddiesenreither In your XCode workspace with your project, select the project and then switch to your target via the dropdown in Xcode. I assume the target will be named `App`. Once you select it , there is a little icon to the left of the dropdown that pulls out a menu. This shows you the `Project` and `Targets`. Double Click in the Target `App` and rename it to `APK-KP`. Let me know how this goes.

**Reply by ddiesenreither** [reply]
Thanks - I already figured out this workaround but the question is still if this is a bug in the run command. Why should i create the Target 'App' if it is not runnable?
